controllers/operandrequest/reconcile_operator.go

```diff
@@ -185,11 +185,21 @@ func (r *Reconciler) createSubscription(ctx context.Context, cr *operatorv1alpha
 	return nil
 }
 
-func (r *Reconciler) updateSubscription(ctx context.Context, cr *operatorv1alpha1.OperandRequest, sub *olmv1alpha1.Subscription) error {
+func (r *Reconciler) updateSubscription(ctx context.Context, cr *operatorv1alpha1.OperandRequest, sub *olmv1alpha1.Subscription, key types.NamespacedName) error {
 
 	klog.V(2).Infof("Updating Subscription %s/%s ...", sub.Namespace, sub.Name)
 	cr.SetUpdatingCondition(sub.Name, operatorv1alpha1.ResourceTypeSub, corev1.ConditionTrue)
 
+	sub.Annotations[key.Namespace+"."+key.Name] = "true"
+	mergePatch, _ := json.Marshal(map[string]interface{}{
+		"metadata": map[string]interface{}{
+			"annotations": sub.Annotations,
+		},
+	})
+	if err := r.Patch(ctx, sub, client.RawPatch(types.MergePatchType, mergePatch)); err != nil {
```

Comment:
Why we patch it oncee and update it once?

Reply:
My bad, I ignore the subscription update itself.
Yes, we could update annotations at line 108 with updates of other `spec` field.